package.json

```diff
@@ -57,39 +57,39 @@
   },
   "private": false,
   "dependencies": {
-    "@angular/animations": "6.1.0",
-    "@angular/cdk": "^6.4.2",
-    "@angular/common": "6.1.0",
-    "@angular/compiler": "6.1.0",
-    "@angular/core": "6.1.0",
-    "@angular/forms": "6.1.0",
-    "@angular/material": "^6.4.2",
-    "@angular/platform-browser": "6.1.0",
-    "@angular/platform-browser-dynamic": "6.1.0",
-    "@angular/router": "6.1.0",
+    "@angular/animations": "7.0.3",
+    "@angular/cdk": "^7.0.3",
+    "@angular/common": "7.0.3",
+    "@angular/compiler": "7.0.3",
+    "@angular/core": "7.0.3",
+    "@angular/forms": "7.0.3",
+    "@angular/material": "^7.0.3",
+    "@angular/platform-browser": "7.0.3",
+    "@angular/platform-browser-dynamic": "7.0.3",
+    "@angular/router": "7.0.3",
     "@ngrx/effects": "6.1.2",
     "@ngrx/router-store": "6.1.2",
     "@ngrx/store": "6.1.2",
     "@ngrx/store-devtools": "6.1.2",
     "core-js": "^2.5.4",
     "geteventlisteners": "^1.0.6",
     "ngrx-store-freeze": "^0.2.4",
-    "office-ui-fabric-react": "6.73.0",
+    "office-ui-fabric-react": "6.110.0",
```

Comment:
Fabric 6.73.0 has some errors when building. These were acknowledged and fixed after that version so I'm updating our version to the latest. Tested locally and everything seems to be working fine.

Reply:
What sort of errors?
Anyway, upgrading is not so simple as you went about here - there are API changes that need to be reflected in our components as `@Input`s etc.

I started doing this myself, not knowing you were working on it too. You can find my changes in the `upgrade-fabric-react` branch (pushed to `origin`), so if you merge it to this branch and solve the conflicts (shouldn't be much) - this can be done in a single PR.

Reply:
Just a few typing issues with the Dropdown and KeytipData type:
![image](https://user-images.githubusercontent.com/41396986/49408239-560c8a80-f710-11e8-848e-59024c42710e.png)

Reply:
Totally didn't realize the API changes were that significant. I'll work on merging your branch and seeing if I can find any other discrepancies

Reply:
Fixed some other deltas. I went through all the components but I wouldn't be surprised if I missed a few properties...